algebird-core/src/main/scala/com/twitter/algebird/CountMinSketch.scala

```diff
@@ -1332,5 +1332,10 @@ object CMSHasherImplicits {
       CMSHasher.hashBytes(a, b, width)(x.getBytes("UTF-8"))
   }
 
+  implicit object CMSHasherBigDecimal extends CMSHasher[BigDecimal] {
+    override def hash(a: Int, b: Int, width: Int)(x: BigDecimal): Int =
+      CMSHasherString.hash(a, b, width)(x.underlying.toPlainString)
```

Comment:
I'm afraid this is slow. Can you compare this to using `x.underlying.scale` and `x.underling.unscaledValue`? You could use the byte array from `unscaledValue` and perhaps multiply the scale by `a` before passing to the `Array[Byte]` hashing.

If this is not faster, we don't have to do it, but allocating strings is generally slow.